packages/atlas/src/components/_nft/NftWidget/NftWidget.styles.ts

```diff
@@ -0,0 +1,136 @@
+import styled from '@emotion/styled'
+
+import { Avatar } from '@/components/Avatar'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgActionJoyToken } from '@/components/_icons'
+import { cVar, sizes } from '@/styles'
+
+type SizeProps = { 'data-size': 'medium' | 'small' }
+
+export const Container = styled.div`
+  background-color: ${cVar('colorCoreNeutral900')};
+  min-width: 0;
+`
+
+export const Content = styled.div<SizeProps>`
+  display: grid;
+  gap: ${sizes(6)};
+  grid-template-columns: 1fr 1fr;
+  justify-content: space-between;
+  box-shadow: inset 2px 0 0 ${cVar('colorCoreBlue500')}, ${cVar('effectDividersTop')}, ${cVar('effectDividersBottom')};
+  padding: ${sizes(6)};
+
+  &[data-size='small'] {
+    grid-template-columns: minmax(0, 1fr);
+    padding: ${sizes(4)};
+    gap: ${sizes(4)};
+  }
+`
+
+export const NFTOwnerContainer = styled.div<SizeProps>`
```

Comment:
This part has a slighlty different height in the design: 
![image](https://user-images.githubusercontent.com/51168865/155733594-a1cc018e-d268-4b38-a1fe-6dd7b5aa70ca.png)

Reply:
It's because I used our button text variant instead of a non-navigable Text component there for the owners name, any objection @toiletgranny?